src/autocomplete/databases/postgresql/tests/extract-statements.test.ts

```diff
@@ -0,0 +1,192 @@
+import {
+    PostgreSqlStatement,
+    extractPostgreSqlStatementsFromQuery,
+} from '../postgresql-extract-statements';
+import {separateQueryAndCursor} from '../../../shared';
+
+test('should extract statements from single query', () => {
+    const query = 'SELECT * FROM art WHERE id = 1;';
+    const result = extractPostgreSqlStatementsFromQuery(query);
+    const expectedResult: PostgreSqlStatement[] = [
+        {
+            startColumn: 1,
+            startLine: 1,
+            endColumn: 31,
+            endLine: 1,
+            statement: 'SELECT * FROM art WHERE id = 1',
```

Comment:
Why don't we add `;`? We'd like to highlight it too, right?

Reply:
no, because bad position of semicolon can break highlighting, look at example
```
SELECT * FROM art




;
```